docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -94,6 +94,14 @@ target: output
 
 A `completionCondition` defines whether the multi-instance body can be completed immediately when the condition is satisfied. It is a [boolean expression](/components/concepts/expressions.md#boolean-expressions) that will be evaluated each time the instance of the multi-instance body completes. Any instances that are still active are terminated and the multi-instance body is completed when the expression evaluates to `true`.
 
+The BPMN 2.0 specification defines the following properties of a multi-instance body instance:
+
+- `numberOfInstances`: The number of instances created.
+- `numberOfActiveInstances`: The number of instances currently active.
+- `numberOfCompletedInstances`: The number of instances already completed.
```

Comment:
❌   The `numberOfTerminatedInstances` are also defined and available for use in the completion condition expression. Let's add it here as well. 

Generally, this value will not work that well in the completion condition, i.e. the completion condition is only evaluated for completing instances, but instances won't complete anymore as soon as the multi-instance is terminating. So, the value will always be 0 when used in the completion condition, simply because of the way the zeebe engine deals with termination. However, the value is available in the completion condition and will take precedence over process variables with the same name. So we should mention it here as well.